components/team_members_dropdown/team_members_dropdown.jsx

```diff
@@ -56,9 +56,7 @@ export default class TeamMembersDropdown extends React.Component {
 
     handleRemoveFromTeam = async () => {
         const {data, error} = await this.props.actions.removeUserFromTeam(this.props.teamMember.team_id, this.props.user.id);
-        if (data) {
-            this.props.actions.getTeamStats(this.props.teamMember.team_id);
```

Comment:
Is this covered by the redux action? We need to update the team stats when a user is removed from the team otherwise the total team member count won't update for the person doing the removing

Reply:
This uses an existing action in webapp which calls for the stats and here we are making another duplicate call so i am removing it..

I changed the existing action name to be more obvious of what it does.